elasticjob-error-handler/elasticjob-error-handler-type/elasticjob-error-handler-email/src/main/java/org/apache/shardingsphere/elasticjob/error/handler/email/EmailJobErrorHandler.java

```diff
@@ -95,28 +96,35 @@ private String createErrorContext(final String jobName, final Throwable cause) {
         return String.format("Job '%s' exception occur in job processing, caused by %s", jobName, writer.toString());
     }
     
-    private Message createMessage(final String content) throws MessagingException {
-        MimeMessage result = new MimeMessage(session);
-        result.setFrom(new InternetAddress(config.getFrom()));
-        result.setSubject(config.getSubject());
+    private Message createMessage(final String content, final EmailConfiguration emailConfiguration) throws MessagingException {
+        MimeMessage result = new MimeMessage(createSession(emailConfiguration));
```

Comment:
Consider using `Optional.ofNullable(session).orElseGet(() -> createSession(emailConfiguration))` to prevent invoking synchronized method everytime.